SteamRPC.js

```diff
@@ -88,8 +97,9 @@ async function loadProfiles() {
             if (typeof profile.title != "string") throw "Exported 'title' couldn't be found or isn't a valid string type!";
             if (typeof profile.appID != "number") throw "Exported 'appID' couldn't be found or isn't a valid number type!";
             if (typeof profile.translateSteamPresence != "function") throw "Exported 'translateSteamPresence' function couldn't be found or isn't a valid function type!";
-            
             if (profiles.hasOwnProperty(profile.appID)) throw `Found two profiles that export the same appID ${profile.appID}! Make sure to change the appID variable in each profile!`;
+
+            updateResources(folder, profile.appID);
```

Comment:
You should await for this, otherwise it might run into weird race conditions.

It also should be placed before the
```js
let profile = await import(`./profiles/${folder}/index.js`);
```
line since with the `import resources from "resources.json";` change I suggested, it'd load the old resources.json I think.